src/Dfe.PlanTech.Web/Configuration/ContactOptions.cs

```diff
@@ -0,0 +1,7 @@
+namespace Dfe.PlanTech.Web.Configuration;
+
+public class ContactOptions
+{
+    public string? Email { get; set; }
```

Comment:
Will this ever use email?

Reply:
Good point! [cc656ae](https://github.com/DFE-Digital/sts-plan-technology-for-your-school/pull/891/commits/cc656ae1ad7d66161c8fb8a38fff1cd9528f1561)